src/toil_scripts/batch_alignment/README.md

```diff
@@ -1,47 +1,146 @@
 ## University of California, Santa Cruz Genomics Institute
-### GATK-compatible Alignment
+### Guide: Running the BWA Pipeline using Toil
+
+This guide attempts to walk the user through running this pipeline from start to finish. If there are any questions
+please contact John Vivian (jtvivian@gmail.com). If you find any errors or corrections please feel free to make a 
```

Comment:
Good catch — Technically this isn't in a script and I thought it'd be useful to provide a point of contact.  We could make a toil-scripts google group and provide that instead?